<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeWidget</class>
 <widget class="QWidget" name="SizeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="userName">
     <property name="text">
      <string>username</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="usrEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passWord">
     <property name="text">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="passEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="sureButton">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
